test/browser.test.js

```diff
@@ -61,6 +61,32 @@ describe('browser support', function() {
     });
   });
 
+  it('loads mixins', function(done) {
+    var appDir = path.resolve(__dirname, './fixtures/browser-app');
+    var options = {
+      appRootDir: appDir,
+      mixinDirs: ['./mixins']
+    };
+
+    browserifyTestApp(options, function(err, bundlePath) {
+      if (err) return done(err);
+
+      var app = executeBundledApp(bundlePath);
+
+      var modelBuilder = (app.registry || app.loopback).modelBuilder;
+
+      var meta = modelBuilder.mixins.meta;
+      var ts = require('./fixtures/browser-app/mixins/time-stamps.json');
```

Comment:
I am proposing to simplify this test to check that `modelBuilder.mixins.TimeStamps` is defined and is a function. That enables us to remove `mixins.meta` as I proposed above.